pydis_site/apps/api/views.py

```diff
@@ -131,7 +131,7 @@ def get(self, request, format=None):  # noqa
             ),
             (
                 "We will not help you with anything that might break a law or the "
-                "terms of service of any other community, site, service, or "
+                "terms of service of any other community, pysite, service, or "
```

Comment:
lol, this can't be right.

Reply:
Oh my god I fixed this like two times.

I guess I was tired. :P